docs/src/taylorize.md

```diff
@@ -208,6 +208,8 @@ list some limitations and advices.
 - Use `local` for internal parameters (simple constant values); this improves
   performance. Do not use it if the variable is Taylor expanded.
 
+- `@taylorize` supports multi-threading via `Threads.@threads`.
```

Comment:
I think we should signal that this is experimental. Do we know if using `@threads` would be thread-safe?

Reply:
> I think we should signal that this is experimental. 

Ok, I'll add that warning.

>  Do we know if using @threads would be thread-safe?

That is up to the user, since that depends on each particular ODE, so we cannot guarantee thread-safety in advance. Perhaps I should document this as well.

Reply:
Yes, it's important to explicit mention it.

Reply:
Ok, both warnings are now documented, thanks!